diagnostics/model_accuracy/verify_inference_model.py

```diff
@@ -87,25 +64,23 @@ def run_inference(model_name, models_dir):
             return model_name, p
 
 
-def check_accuracy(model, p, tolerance=0.001):
+def check_accuracy(model, p, json_file_name, tolerance=0.001):
     #check if the accuracy of the model inference matches with the published numbers
     #Accuracy values picked up from here https://github.com/tensorflow/models/tree/master/research/slim
-    accuracy = \
-    '[{"model_name" : "Inception_v4", "accuracy" : "0.802"},\
-     {"model_name" : "ResNet50_v1", "accuracy" : "0.752"}, \
-     {"model_name" : "MobileNet_v1", "accuracy" : "0.709"}]'
-
-    data = json.loads(accuracy)
 
+    data = parse_json(args.json_file_name)
```

Comment:
We could just return the `data` which we already parse in line#45 from `run_inference` function and use it here, instead of parsing it here again in `check_accuracy` function.

Reply:
This can be done in a later PR.